### ISSUE TITLE
New feature request: copied sound response.

### ISSUE THREAD

**Issue by FaiChou** [Clipy/Clipy#389]
I noticed [Paste](https://pasteapp.io/) has a feature: when clipboard receive new content, there's a sound response to notice user.

When copied something on iOS, the computer(macOS) will receive the content by `Handoff`,  so it will response a sound.

**Comment by codeshif** [comment]
good idea, you can use AVPlayer implement it, but it's like you need to make a property in the settings panel.
